src/app.module.ts

```diff
@@ -1,10 +1,19 @@
 import { Module } from "@nestjs/common";
+import { ConfigModule } from "@nestjs/config";
+import { TypeOrmModule } from "@nestjs/typeorm";
 
 import { AppController } from "./app.controller";
 import { AppService } from "./app.service";
+import { dataSourceOptions } from "./config/data-source";
+import { validate } from "./config/env.validation";
 
 @Module({
-  imports: [],
+  imports: [
+    ConfigModule.forRoot({
+      validate,
+    }),
+    TypeOrmModule.forRoot(dataSourceOptions),
```

Comment:
may be able to use async loading to inject config service, see below.

Reply:
I only use ConfigModule to validate the envariables

But the TypeOrmModule
See [src/config/data-source.ts](https://github.com/Cophr/backend/pull/2/files/83c3e938a7537db023fdd0a059d17bb03b4e8034#diff-6a2cdd98dafffa3a3c5bf5f5fe257b0adbb57dbfe57d93b2297b3fff1dbd3e2d)

Setting two database connection I think not well.

Reply:
Okay, we got this situation in above conversation, looks good now.
btw, I'm not meant to set up two connections.